Core/include/Acts/Seeding/detail/CylindricalSpacePointGrid.ipp

```diff
@@ -185,36 +195,17 @@ void Acts::CylindricalSpacePointGridCreator::fillGrid(
     float spY = sp.y();
     float spZ = sp.z();
 
-    // store x,y,z values in extent
-    rRangeSPExtent.extend({spX, spY, spZ});
-
     // remove SPs according to experiment specific cuts
     if (!config.spacePointSelector(sp)) {
       continue;
     }
 
-    // remove SPs outside z and phi region
-    if (spZ > config.zMax || spZ < config.zMin) {
-      continue;
-    }
-
-    float spPhi = std::atan2(spY, spX);
-    if (spPhi > config.phiMax || spPhi < config.phiMin) {
-      continue;
-    }
-
-    // calculate r-Bin index and protect against overflow (underflow not
-    // possible)
-    std::size_t rIndex =
-        static_cast<std::size_t>(sp.radius() / config.binSizeR);
-    // if index out of bounds, the SP is outside the region of interest
-    if (rIndex >= numRBins) {
-      continue;
-    }
+    // store x,y,z values in extent
+    rRangeSPExtent.extend({spX, spY, spZ});
```

Comment:
How does this related to https://github.com/acts-project/acts/pull/3688?

Reply:
it will clash. This has to be removed from here. The extend is totally pointless